simbad/parsers/anode_parser.py

```diff
@@ -0,0 +1,37 @@
+"""Returns values for the top anomalous peak from a anode lsa file"""
+
+__author__ = "Adam Simpkin"
+__date__ = "13 April 2018"
+__version__ = "0.1"
+
+import simbad.parsers
+
+
+class AnodeParser(simbad.parsers._Parser):
+    """Class to mine information from a anode lsa"""
+
+    def __init__(self, lsa_file):
+        super(AnodeParser, self).__init__(lsa_file)
+
+        self.x = None
+        self.y = None
+        self.z = None
+        self.peak_height = None
+        self.nearest_atom = None
+
+        self._parse(lsa_file)
+
+    def _parse(self, lsa_file):
```

Comment:
why do some parsers define this function as `parse` and others as `_parse` ... keep it the same